charts/spire/charts/spire-server/templates/statefulset.yaml

```
@@ -51,8 +51,10 @@ spec:
       initContainers:
       {{- if and .Values.upstreamAuthority.certManager.enabled .Values.upstreamAuthority.certManager.ca.create }}
         - name: wait
+          {{- with .Values.tools.kubectl.image.securityContext }}
```

```suggestion
          {{- with .Values.tools.kubectl.securityContext }}
```

should be one level up.